apps/emqx_resource/include/emqx_resource.hrl

```diff
@@ -43,3 +43,4 @@
 -type after_query_fun() :: {fun((...) -> ok), Args :: [term()]}.
 
 -define(TEST_ID_PREFIX, "_test_:").
+-define(DEFAULT_RESOURCE_GROUP, <<"default">>).
```

Comment:
If the only purpose of this macro is for testing, we can defined it in the test cases.

Reply:
I will move it to the test file.